x-pack/plugins/cases/common/index.ts

```diff
@@ -15,16 +15,25 @@
 // For example, constants below could eventually be in a "kbn-cases-constants" instead.
 // See: https://docs.elastic.dev/kibana-dev-docs/key-concepts/platform-intro#public-plugin-api
 
-export type { Case, Cases, CasesBulkGetResponse } from './api';
+export type {
+  CasesBulkGetResponse,
+  CasePostRequest,
+  GetRelatedCasesByAlertResponse,
+  UserActionFindResponse,
+} from './types/api';
+export type { Case, Cases, RelatedCase } from './types/domain';
 export type {
   CaseUI,
   CasesUI,
   CasesFindResponseUI,
   Ecs,
   CaseViewRefreshPropInterface,
   CasesPermissions,
+  CasesStatus,
 } from './ui/types';
 
+export { CaseSeverity, CaseStatuses } from './types/domain';
```

Comment:
Follow up to another question I asked, do we need `CaseStatuses` here if it is already exported in `cases-components`? The fewer things we have here the better the page load bundle size will be.

Reply:
Good point! I will remove it.